### PR TITLE
Added sto4_start_time column to be initialized on unit creation

### PR DESCRIPTION
**Pull request by rv0lt**
## 1. Description / Summary

Now the column _Sto4_Start_time_ will automatically be set in the _Units_ table when the create-unit command is run. Instead of having to run the upgrade unit.

## 2. Jira task / GitHub issue

https://scilifelab.atlassian.net/jira/software/projects/DDS/boards/13?selectedIssue=DDS-1668

## 3. Type of change

What _type of change(s)_ does the PR contain?

**Check the relevant boxes below. For an explanation of the different sections, enter edit mode of this PR description template.**

- [ ] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [ ] Non-breaking 
- [ ] Database change: _Remember the to include a new migration version, **or** explain here why it's not needed._ 
- [x] Bug fix 
- [ ] Security Alert fix 
- [ ] Documentation 
- [ ] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [x] [Sprintlog](../SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [ ] PR to `master` branch: \_If checked, read [the release instructions](../doc/procedures/new_release.md) 
  - [ ] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](../.github/pull_request_template.md).**

- [x] **Black**

- [x] **Prettier**

- [x] **Yamllint**

- [x] **Tests**

- [x] **CodeQL**

- [x] **Trivy**

- [x] **Snyk**

<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1469"></a>

### CODE REVIEW

**Review by i-oden — CHANGES_REQUESTED** [review]
Also we'll need to change the test `dds_web/tests/test_commands.py::test_create_new_unit_success` so that it checks that the `sto4_start_time` variable is set. In this case it would be good if you checked that only all `sto4*` variables were set and not `sto2*`.

**Comment by rv0lt** [comment]
@i-oden Should I also add checks for the rest of the parameters now that I am modiying this test. As right now only checks that the name was correct ?

**Comment by i-oden** [comment]
> @i-oden Should I also add checks for the rest of the parameters now that I am modiying this test. As right now only checks that the name was correct ?

Yeah, you can do that, thanks

File: tests/test_commands.py [hunk 1]
```diff
@@ -252,6 +252,26 @@ def test_create_new_unit_success(client, runner, capfd: LogCaptureFixture) -> No
...
+    assert new_unit.sto4_start_time
+    assert new_unit.sto4_endpoint == correct_unit["safespring_endpoint"]
+    assert new_unit.sto4_name == correct_unit["safespring_name"]
+    assert new_unit.sto4_access == correct_unit["safespring_access"]
+    assert new_unit.sto4_secret == correct_unit["safespring_secret"]
+    assert new_unit.days_in_available
+    assert new_unit.days_in_expired
+    assert new_unit.quota == correct_unit["quota"]
+    assert new_unit.warning_level
+
```
Comment (i-oden) [inline tests/test_commands.py:274]: Also verify that `sto2_endpoint`,  `sto2_name`, `sto2_access` and `sto2_secret` are not set

**Review by i-oden — CHANGES_REQUESTED** [review]

File: tests/test_commands.py (hunk 1 above)
Comment (i-oden) [reply tests/test_commands.py:274]: They shouldn't be ofc but add it anyway

Comment (i-oden) [reply tests/test_commands.py:274]: Actually skip this. It's unneccessary and it should be tested in `test_models` if anything

Comment (rv0lt) [reply tests/test_commands.py:274]: Oops Oki, I will go back

Comment (i-oden) [reply tests/test_commands.py:274]: Sorry 😅

**Review by i-oden — APPROVED** [review]
Looks good!

**Comment by i-oden** [comment]
@rv0lt just rerun the tests

**Pull request merged**